webapp/components/wpt-results.html

```diff
@@ -326,6 +368,17 @@
         this.refreshDisplayedNodes();
       }
 
+      async fetchManifest(path) {
+        if (!this.computePathIsATestFile(path)) {
+          return;
+        }
+        this.manifest = null;
+        const that = this;
```

Comment:
Per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions, `that = this` ought not be needed. And because we're in an async function this can use `await` instead of `then()` to not have the arrow function at all, avoiding the issue.